docs/UserGuide.md

```diff
@@ -90,7 +81,20 @@ Beyond individual products, you can also:
 
 ### 2.4 User Interface
 
-{ replace with a diagram of MyInsuRec and arrows and notes describing the different components of MyInsuRec }
+*MyInsuRec* comes with a simple [GUI](#gui) to help you see your clients and meetings easily.
+The user interface consists of a main window, command result window, [CLI](#cli),and toolbar.
+
+Below the GUI title, you will see a toolbar which can be used to change light/dark mode, open the help window, and exit the app.
+
+The main window shows the list of clients, meetings or products on executing a list command.
+It also shows a lists of details associated with a client, meeting, or product on executing a view command.
+
+All commands are accepted from the user via the [CLI](#cli).
+
+The result of executing a command is shown in the command result window just above the CLI.
+Here you will see help, command success, and warning messages.
+
+![Ui](images/annotatedGui.png)
```

Comment:
Hi I think is portion should be done by me. Not sure why you had this filled in as well

Reply:
my bad, didn't check that it was assigned already 😅